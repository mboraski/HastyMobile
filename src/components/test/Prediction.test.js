// TODO: Remove when tests implemented
test.only('real recognize real', () => {
    expect('real').toBe('real');
});
// import React from 'react';
// import 'react-native';
// import { shallow } from 'enzyme';
// import Component from '../Prediction';
//
// describe('Prediction', () => {
//     it('renders correctly', () => {
//         const props = {
//             prediction: {
//                 description: 'description'
//             }
//         };
//         const render = shallow(<Component {...props} />);
//         expect(render).toMatchSnapshot();
//     });
// });
