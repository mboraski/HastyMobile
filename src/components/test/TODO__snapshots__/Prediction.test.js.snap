// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Prediction renders correctly 1`] = `
<TouchableOpacity
  activeOpacity={0.2}
  style={
    Array [
      Object {
        "alignItems": "center",
        "borderBottomWidth": 0.5,
        "borderColor": "#cccccc",
        "flexDirection": "row",
        "paddingHorizontal": 35,
        "paddingVertical": 14.4,
      },
      undefined,
    ]
  }
>
  <Image
    source={1}
    style={
      Object {
        "height": 32,
        "marginRight": 15,
        "width": 32,
      }
    }
  />
  <Text
    accessible={true}
    allowFontScaling={true}
    disabled={false}
    ellipsizeMode="tail"
    style={
      Object {
        "flex": 1,
        "flexWrap": "wrap",
        "fontSize": 16,
      }
    }
  >
    description
  </Text>
</TouchableOpacity>
`;
